### PR TITLE
#46 Fix regexp support for resource names

### PR DESCRIPTION
**Pull request by vincent-zurczak**
Fix the behavior for resource names to match what was written in the README file.

### CODE REVIEW

**Review by brokenpip3 — APPROVED** [review]

**Comment by vincent-zurczak** [comment]
Hello @brokenpip3.
Thank you for your review. However, the PR is still blocked as « approving review is required by reviewers with write access ». Was there a change recently?

**Comment by brokenpip3** [comment]
Probably I do not have anymore the write grants in this repo :)

**Comment by martin-schulze-vireso** [comment]
Yes, the permissions granted by the homebrew team were initially too broad. If you want brokenpip3 as co-maintainer, we can add him to the bats-detik team.

File: lib/detik.bash [hunk 1]
```diff
@@ -380,3 +386,21 @@ build_k8s_client_with_options() {
...
+
+
+# Filters results by resource name (or name pattern).
+# The results are directly read, they are not passed as variables.
+#
+# @param $1 the resource name or name pattern
+# @return 0
+filter_by_resource_name() {
+
+	# For all the output lines...
```
Comment (martin-schulze-vireso) [inline lib/detik.bash:398]: This feels like it could be a single regex grep. If not, we should at least quote the parameter

**Review by martin-schulze-vireso — APPROVED** [review]

File: lib/detik.bash (hunk 1 above)
Comment (vincent-zurczak) [reply lib/detik.bash:398]: Hello.
The pattern must really apply to the first column only.
I have quoted the parameter.

**Comment by vincent-zurczak** [comment]
If @brokenpip3 agrees, I see no objection to add him in the bats-detik team.

**Comment by brokenpip3** [comment]
>If @brokenpip3 agrees, I see no objection to add him in the bats-detik team.

my pleasure :)

**Pull request merged**